cinema_service/urls.py

```diff
@@ -5,4 +5,5 @@
     path("admin/", admin.site.urls),
     path("api/cinema/", include("cinema.urls", namespace="cinema")),
```

Comment:
The `include` function uses the `namespace` argument for the `cinema` app. Ensure that the `cinema/urls.py` file defines an `app_name` variable to support this namespacing.